### PR TITLE
Fix celery autoreload

### PR DESCRIPTION
**Pull request by EnTeQuAk**
r? @kumar303 @diox 

Works perfectly here, restarts on any python-file changes for now

```
==> /code/logs/docker-celery.log <==
on_any_event(self=<watchdog.tricks.AutoRestartTrick object at 0x2c5d6d0>, event=<FileModifiedEvent: src_path='/code/src/olympia/activity/tasks.py'>)

worker: Hitting Ctrl+C again will terminate all running tasks!

worker: Warm shutdown (MainProcess)
```

but that should be fine imho.

### CODE REVIEW

**Comment by muffinresearch** [comment]
@EnTeQuAk might be an idea to double check with ops  re: `configs/init.d/celery` being unused just in-case.

**Comment by EnTeQuAk** [comment]
> @EnTeQuAk might be an idea to double check with ops re: configs/init.d/celery being unused just in-case.

Answer:
> 15:53 <jason> cgrebs: nope!

**Comment by EnTeQuAk** [comment]
going to debug this circleci failure a bit, seems related to the autoreload change.

File: docker/supervisor-celery.conf [hunk 1]
```diff
@@ -7,17 +7,18 @@ logfile=/code/logs/supervisord-celery.log
 # https://github.com/mozilla-services/puppet-config/blob/master/amo/modules/celery/manifests/service.pp#L20
 # Changes:
 # - Set concurrency (number of workers) back to the default
-# - The code auto-reloads for development
-command=celery -A olympia.amo.celery:app worker --autoreload -E --loglevel=INFO
+command=watchmedo auto-restart -d /code/ -p '*.py' -R celery -- -A olympia.amo.celery:app worker -E --loglevel=INFO
```
Comment (kumar303) [inline docker/supervisor-celery.conf]: This looks like the right thing to do but we have [had issues](https://forums.docker.com/t/file-access-in-mounted-volumes-extremely-slow-cpu-bound/8076) with file watching in the past. Just something to be aware of.

Comment (EnTeQuAk) [inline docker/supervisor-celery.conf]: Good point, I couldn't see any massive performance implications on my linux machine but it'd be nice to get some testing of this by someone using a docker-machine setup before we'll merge this.

**Comment by EnTeQuAk** [comment]
Closing for now untill I have time to test this further and debug circleci failures.

**Pull request closed without merging**